js/gameConfig.js

```diff
@@ -25,96 +25,74 @@ const units = {
-    'Knight': {name: 'Knight', maxXp: 100000, energyGeneration: 300},
-    'Veteran knight': {name: 'Veteran knight', maxXp: 1000000, energyGeneration: 1000},
-    'Elite knight': {name: 'Elite knight', maxXp: 7500000, energyGeneration: 3000},
-    'Holy knight': {name: 'Holy knight', maxXp: 40000000, energyGeneration: 15000},
-    'Legendary knight': {name: 'Legendary knight', maxXp: 150000000, energyGeneration: 50000},
-
-    'Student': {name: 'Student', maxXp: 100000, energyGeneration: 100},
-    'Apprentice mage': {name: 'Apprentice mage', maxXp: 1000000, energyGeneration: 1000},
-    'Mage': {name: 'Mage', maxXp: 10000000, energyGeneration: 7500},
-    'Wizard': {name: 'Wizard', maxXp: 100000000, energyGeneration: 50000},
-    'Master wizard': {name: 'Master wizard', maxXp: 10000000000, energyGeneration: 250000},
-    'Chairman': {name: 'Chairman', maxXp: 1000000000000, energyGeneration: 1000000},
+    Destroyer: {name: 'Destroyer', maxXp: 50, maxLayers: 5, progressBarId: 'battleProgressBar'},
 };
 
 const skillBaseData = {
-    'Concentration': {name: 'Concentration', maxXp: 100, effect: 0.01, description: 'Research xp'},
-    'Productivity': {name: 'Productivity', maxXp: 100, effect: 0.01, description: 'Module xp'},
-    'Bargaining': {name: 'Bargaining', maxXp: 100, effect: -0.01, description: 'Energy usage'},
-    'Meditation': {name: 'Meditation', maxXp: 100, effect: 0.01, description: 'Population'},
-
-    'Strength': {name: 'Strength', maxXp: 100, effect: 0.01, description: 'Military energy generation'},
-    'Battle tactics': {name: 'Battle tactics', maxXp: 100, effect: 0.01, description: 'Military xp'},
-    'Muscle memory': {name: 'Muscle memory', maxXp: 100, effect: 0.01, description: 'Strength xp'},
-
-    'Mana control': {name: 'Mana control', maxXp: 100, effect: 0.01, description: 'T.A.A. xp'},
-    'Immortality': {name: 'Immortality', maxXp: 100, effect: 0.01, description: 'Longer lifespan'},
-    'Time warping': {name: 'Time warping', maxXp: 100, effect: 0.01, description: 'Gamespeed'},
-    'Super immortality': {name: 'Super immortality', maxXp: 100, effect: 0.01, description: 'Longer lifespan'},
-
-    'Dark influence': {name: 'Dark influence', maxXp: 100, effect: 0.01, description: 'All xp'},
-    'Evil control': {name: 'Evil control', maxXp: 100, effect: 0.01, description: 'Evil gain'},
-    'Intimidation': {name: 'Intimidation', maxXp: 100, effect: -0.01, description: 'Energy usage'},
-    'Demon training': {name: 'Demon training', maxXp: 100, effect: 0.01, description: 'All xp'},
-    'Blood meditation': {name: 'Blood meditation', maxXp: 100, effect: 0.01, description: 'Evil gain'},
-    'Demon\'s wealth': {name: 'Demon\'s wealth', maxXp: 100, effect: 0.002, description: 'Energy generation'},
-
+    Concentration: {name: 'Concentration', maxXp: 100, effects: [{effectType: EffectType.Population, baseValue: 5}, {effectType: EffectType.Energy, baseValue: 5}]},
```

Comment:
```suggestion
    Concentration: {name: 'Concentration', maxXp: 100, effects: [{type: EffectType.Population, baseValue: 5}, {type: EffectType.Energy, baseValue: 5}]},
```
Einfach weil die ja schon Teil der effects sind 🙃